generator/generate.js

```diff
@@ -0,0 +1,20 @@
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const fs = require('fs-extra');
+
+async function generateDeclarations() {
+    const {stdout, stderr} = await exec('node_modules\\.bin\\ng-swagger-gen --config ' +
+        '.\\generator\\config\\ng-swagger-gen.json');
```

Comment:
![obraz](https://media.giphy.com/media/yYZOdU9PjcIDK/giphy.gif)

Reply:
Wszędzie indziej są `/` w sumie

Reply:
Plus formater tutaj łamie linijkę, jak się go poprosi, bo mamy w jsie długość linii na 80

Reply:
W ogóle wszystkie jsy formatuje inaczej

Reply:
linuksowe slesze nie działały u mnie

Reply:
Formater krzyczy, bo nie ma formatera do jsa
A jak widziałem zalecane " zamiast ' to postanowiłem go olać

Reply:
Meh, nie znam się na nołdzie, ale chyba trzeba będzie ustalić na jakim systemie to będzie odpalane

Reply:
A skoro mamy #teamsiszarp, to...

Reply:
Jest do jsa, coś tam do wszystkiego IntelliJ dorzuca

https://github.com/papaya-agh/papaya-frontend/blob/6b8dcaba1a1b9cf31e5c085410569975fc133c03/.idea/codeStyles/Project.xml#L241

Reply:
> Meh, nie znam się na nołdzie, ale chyba trzeba będzie ustalić na jakim systemie to będzie odpalane

Eee, na pewno jest sposób, komputery są mądre. Też się nie znam, ale mogę coś poszukać/popytać. Jeśli chodzi o system, to nie wiem, jak by się na to zapatrywało CI/Heroku plus DOBRY KOD DZIAŁA WSZĘDZIE

Reply:
@maciekak Ty coś mówiłeś, że kojarzysz JSa, to to jest niezły moment na wykazanie się

Reply:
coś sobie ~popsułem~  naprawiłem i już nie pokazuje mi złego formatowania
pytanie tylko, jak też u was to ~popsuć~ naprawić

Reply:
> coś sobie popsułem naprawiłem i już nie pokazuje mi złego formatowania
> pytanie tylko, jak też u was to popsuć naprawić

Jak ~naprawiłeś~ popsułeś w ustawieniach IntelliJa, to wystarczy wyeksportować xmla od nowa - to sobie siedzi tam pod trybikiem. Tam już lecą te wszystkie języki razem, także wystarczy nadpisać aktualnego xmla

Reply:
https://stackoverflow.com/questions/38134548/nodejs-exec-command-for-both-windows-and-ubuntu

Reply:
już ogarnąłem, sprawdź jeśli możesz

Reply:
Śmiga